contracts/interfaces/IAxelarValuedExpressExecutableWithToken.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { IAxelarExpressExecutableWithToken } from './IAxelarExpressExecutableWithToken.sol';
+import { IAxelarValuedExpressExecutable } from './IAxelarValuedExpressExecutable.sol';
+
+/**
+ * @title IAxelarValuedExpressExecutableWithToken
+ * @dev Interface for the Axelar Valued Express Executable With Token contract.
+ */
+interface IAxelarValuedExpressExecutableWithToken is IAxelarExpressExecutableWithToken, IAxelarValuedExpressExecutable {
+    /**
+     * @dev Returns the value (token address and amount) associated with a contract call with token.
+     * @param sourceChain The source blockchain.
```

Comment:
Haven't seen the word `blockchain` in the code. I guess just `source chain` would be enough
```suggestion
     * @param sourceChain The source chain.
```